rivers.py

```
@@ -0,0 +1,302 @@
+    'INPUT': output_path,
+    'SOURCE_CRS': QgsCoordinateReferenceSystem('EPSG:4326'),
+    'TARGET_CRS': QgsCoordinateReferenceSystem('EPSG:3857'),
+    'RESAMPLING': 0, 'NODATA': -9999, 'TARGET_RESOLUTION': 30,
+    'OPTIONS': '', 'DATA_TYPE': 0, 'TARGET_EXTENT': None,
+    'TARGET_EXTENT_CRS': None, 'MULTITHREADING': False,
+    'EXTRA': '', 'OUTPUT': 'TEMPORARY_OUTPUT'})['OUTPUT']
+
+# Загрузка алгоритмов SAGA
+provider = SagaNextGenAlgorithmProvider()
+provider.loadAlgorithms()
+QgsApplication.processingRegistry().addProvider(provider=provider)
```

Ruff (0.7.0)

Undefined name `QgsApplication`

(F821)